### PR TITLE
Created Zabbix reporter

### PR DESCRIPTION
**Pull request by ljelinkova**

### CODE REVIEW

File: ee_tests/cico_run_EE_tests_ts.sh [hunk 1]
```diff
@@ -7,6 +7,9 @@
 # $5 = quickstart name
 # $6 = release strategy
 # $7 = feature level
+# $8 = enable zabbix reporting
+# $9 = zabbix host
+# $10 = zabbix metric prefix
```
Comment (ppitonak) [inline ee_tests/cico_run_EE_tests_ts.sh]: We agreed that we wouldn't introduce new property on top-level for zabbix prefix, we would just reuse feature_level

File: ee_tests/cico_run_EE_tests_ts.sh [hunk 2]
```diff
@@ -28,6 +31,12 @@ RELEASE_STRATEGY=${6:-$DEFAULT_RELEASE_STRATEGY}
 DEFAULT_FEATURE_LEVEL="released"
 FEATURE_LEVEL=${7:-$DEFAULT_FEATURE_LEVEL}
 
+ZABBIX_ENABLED=${8:-"true"}
+
+ZABBIX_HOST=${9:-"unknown"}
+
+ZABBIX_METRIC_PREFIX=${10:-"unknown"}
```
Comment (ppitonak) [inline ee_tests/cico_run_EE_tests_ts.sh]: not necessary, see previous comment

File: ee_tests/src/quickstart_deployments.spec.ts [hunk 3]
```diff
@@ -53,13 +53,13 @@ describe('Main E2E test suite', () => {
...
 
-  it('Login', async () => {
+  it('login', async () => {
     support.info('--- Login ---');
     let login = new support.LoginInteraction();
     await login.run();
   });
 
-  it('Check feature level', async () => {
+  it('feature-level', async () => {
```
Comment (ppitonak) [inline ee_tests/src/quickstart_deployments.spec.ts]: I think that underscores make the string more readable but you can leave it like this

File: ee_tests/src/support/zabbix_reporter.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            this.getDuration(result.status);
+        this.records.push(record);
+    }
+
+    public suiteDone(result: CustomReporterResult) {
+        if (!fs.existsSync(this.OUTPUT_DIRECTORY)) {
+            fs.mkdirSync(this.OUTPUT_DIRECTORY);
+        }
+
+        let stream = fs.createWriteStream(this.OUTPUT_DIRECTORY + this.suiteName + '-zabbix-report.txt');
```
Comment (ppitonak) [inline ee_tests/src/support/zabbix_reporter.ts]: Wouldn't it be easier if the file name didn't include suite name (later when we start sending the result to Zabbix)?

**Comment by ljelinkova** [comment]
I agree with all comments and update the PR.

**Pull request merged**